README.md

```diff
@@ -17,7 +17,17 @@ npm install --save @codaco/ui
 }
 ```
 
-## Usage
+## Development
+
+In this repo:
+
+`npm link` then in consuming app ``
+
+After each change:
+
+`npm install; npm run build; npm install --only=production`
```

Comment:
Should just be able to do a build in the local UI directory (which will now be symlinked), unless there are packages that have changed?

Reply:
Ah. This was to do with the react hooks errors I was getting when I tried this. The above command was one you suggested, and it has been the only thing that works for me!

Reply:
To be clear: this is only when I am developing on UI and importing via npm link into NC - it could be different for Architect maybe?